Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Service/ReportService.cs

```diff
@@ -17,17 +20,20 @@ public class ReportService : IReportService
 		public static readonly Log Log = Log.Instance;
 		private readonly string _communityFolderPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "SDL Community", "ExportAnalysisReports");
 		private readonly IMessageBoxService _messageBoxService;
-		private readonly IProjectService _studioService;
+		private readonly IProjectService _projectService;
+		private string _reportFile;
+		private IEnumerable<AnalyzedFile> _analyzedFiles;
 
 		public ReportService(IMessageBoxService messageBoxService, IProjectService studioService)
 		{
 			_messageBoxService = messageBoxService;
-			_studioService = studioService;
+			_projectService = studioService;
```

Comment:
Change "studioService" to "projectService"

Reply:
The code was updated, see 
![image](https://user-images.githubusercontent.com/13149323/81561646-2ac5aa80-939c-11ea-810a-470a0a8cdcdf.png)